definitions/npm/@storybook/addon-a11y_v3.x.x/test_addon-a11y_v3.x.x.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import { describe, it } from 'flow-typed-test';
+import React from 'react';
+
+// $ExpectError
```

Comment:
What's this for?

Reply:
Without the comment:
```
Cannot resolve module `@storybook/react`.

import { storiesOf } from '@storybook/react';
                           ^^^^^^^^^^^^^^^^^^
```

Without the import:
```
Cannot resolve name `storiesOf`.

storiesOf('button', module)
^^^^^^^^^
```

Choose your poison :)

Reply:
Look above ;)

Reply:
So what's the desired update then? @AndrewSouthpaw @GAntoine